src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -357,11 +357,11 @@ public final class JavadocTokenTypes {
      * <pre>{@code @myJavadocTag some magic}</pre>
      * <b>Tree:</b>
      * <pre>{@code
-     *   |--JAVADOC_TAG[3x0] : [@myJavadocTag some magic]
-     *       |--CUSTOM_NAME[3x0] : [@myJavadocTag]
-     *       |--WS[3x13] : [ ]
-     *       |--DESCRIPTION[3x14] : [some magic]
-     *           |--TEXT[3x14] : [some magic]
+     *   JAVADOC_TAG --> JAVADOC_TAG
+     *       |--CUSTOM_NAME --> @myJavadocTag
```

Comment:
This was done in previous PR, please remove it from PR.

This PR should have changes only for |--JAVADOC_INLINE_TAG_START